File: BasicInputOutput/Java/FahrenheitToCelsius.java

import java.util.Scanner;

class FahrenheitToCelsius{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter the Fahrenheit value ; ");
        double Fahrenheit = sc.nextDouble();

        System.out.println("Celsius : "+(((Fahrenheit-32)*5)/9));
    }
}

File: BasicInputOutput/C++/FahrenheitToCelsius.cpp
#include <iostream>
using namespace std;

int main() {
    double fahrenheit, celsius;
    cout << "Enter the Fahrenheit value: ";
    cin >> fahrenheit;

    celsius = ((fahrenheit - 32) * 5) / 9;
    cout << "Celsius: " << celsius << endl;

    return 0;
}
